server/schedule/operator/operator.go

```diff
@@ -80,9 +80,12 @@ type StoreInfluence struct {
 }
 
 // ResourceSize returns delta size of leader/region by influence.
-func (s StoreInfluence) ResourceSize(kind core.ResourceKind) int64 {
+func (s StoreInfluence) ResourceSize(kind core.ResourceKind, opt ...core.LeaderScheduleKind) int64 {
```

Comment:
How about rename it to `resourceScore`? `size` is not accurate any more because it may be count rather than size.